client-src/default/index.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+// webpack@5 doesn't inject node polyfill automatically
+window.global = window.global || window;
```

Comment:
this was needed to make the other polyfills work. Is this way to handle this okay, or is it a problem that this is ignoring webpack's node option https://webpack.js.org/configuration/node/ ?

Any suggestions?

Reply:
Hm, why we need `process` and `util`?

Reply:
> Hm, why we need process and util?

@evilebottnawi webpack5 tests were not passing, this made them work. Not sure if there is a better solution. @hiroppy Originally put that in here:
https://github.com/webpack/webpack-dev-server/commit/b6af4e96c59d8c3cecae534c2b9f7fb8ade8c4e9#diff-bd664d218d585ff7bd6ceced46bd877aR3

Reply:
We can use `alias` here when we will switch on `webpack@5`, we really need full `process/browser` and `util/util` packages? Maybe we can copy post some function to avoid using full packages?

Reply:
do you mean resolve.alias? https://webpack.js.org/configuration/resolve/#resolvealias